packages/next/src/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -292,8 +292,8 @@ export class TraceEntryPointsPlugin implements webpack.WebpackPluginInstance {
           // include the client reference manifest
           const clientManifestsForPage =
             entrypoint.name.endsWith('/page') ||
-            entrypoint.name === '/not-found' ||
-            entrypoint.name === '/_not-found'
+            entrypoint.name === 'app/not-found' ||
+            entrypoint.name === 'app/_not-found'
```

Comment:
Can we add a test when `global.isNextStart` to ensure this is added to the `.nft.json` file?